index.html

```diff
@@ -0,0 +1,301 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <title>Website Design Project</title>
+        <link rel="stylesheet" href="css/style.css" type="text/css">
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+
+    </head>
+    <body>
+        <header class="imagecontainer">
+            <div class="defineimage"></div>
+            <img src="images/ic_menu.svg" alt="menu" onclick="mobile_menu()">
+        </header>
+        <div class="horizontal_line">
+            <a href="#text1" class="wrap ite1">Hello</a>
+            <a href="#text3" class="wrap ite2">Portfolio</a>
+            <a href="#abouttext" class="wrap ite4">About</a>
+            <a href="#text10" class="wrap ite5">Contact</a>
+        </div>
```

Comment:
Kindly use the ul tag and style the items as shown in the original design below
### Original design
![right](https://user-images.githubusercontent.com/58376020/127936467-8cf4151d-4ad7-4d5c-9fad-323ae2f91d83.png)

### Your Project
![wrong](https://user-images.githubusercontent.com/58376020/127936484-9c358ed8-9140-4b40-8ed8-6a03e4c234d8.png)

Reply:
I have made the changes as per the reviewer's suggestions. The shown picture here is the menu design coded in javascript. And the ul, li elements are used in javascript coding. Kindly refer to my javascript code. I have finished the changes and updated the code.